derp/derp_server.go

```diff
@@ -466,10 +470,23 @@ func (c *sclient) setPreferred(v bool) {
 		return
 	}
 	c.preferred = v
+	var homeMove *expvar.Int
 	if v {
 		c.s.curHomeClients.Add(1)
+		homeMove = &c.s.homeMovesIn
 	} else {
 		c.s.curHomeClients.Add(-1)
+		homeMove = &c.s.homeMovesOut
+	}
+
+	// Keep track of varz for home serve moves in/out.  But ignore
+	// the initial packet set when a client connects, which we
+	// assume happens within 5 seconds. In any case, just for
+	// graphs, so not important to miss a move. But it shouldn't:
+	// the netcheck/re-STUNs in magicsock only happen about every
+	// 30 seconds.
+	if time.Since(c.connectedAt) > 5*time.Second {
```

Comment:
As the comment says, this will miss moves in a bunch of cases. If you're happy with that, I don't mind, just rechecking :)

Reply:
I think this will almost never miss anything, actually.

Reply:
(per chat: this is only for steady-state flops)